Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Driver/Version1/IndexerDriver.php

```diff
@@ -0,0 +1,162 @@
+<?php
+namespace Flowpack\ElasticSearch\ContentRepositoryAdaptor\Driver\Version1;
+
+/*
+ * This file is part of the Flowpack.ElasticSearch.ContentRepositoryAdaptor package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Flowpack\ElasticSearch\ContentRepositoryAdaptor\Driver\AbstractDriver;
+use Flowpack\ElasticSearch\ContentRepositoryAdaptor\Driver\DriverInterface;
+use Flowpack\ElasticSearch\ContentRepositoryAdaptor\Driver\IndexerDriverInterface;
+use Flowpack\ElasticSearch\ContentRepositoryAdaptor\Mapping\NodeTypeMappingBuilder;
+use Flowpack\ElasticSearch\Domain\Model\Document as ElasticSearchDocument;
+use TYPO3\Flow\Annotations as Flow;
+use TYPO3\TYPO3CR\Domain\Model\NodeInterface;
+
+/**
+ * Fulltext Indexer Driver for Elastic version 1.x
+ *
+ * @Flow\Scope("singleton")
+ */
+class IndexerDriver extends AbstractDriver implements IndexerDriverInterface
```

Comment:
@kdambekalns Can you check if your recent fixes in the Groovy script are corectly ported here ?